### PR TITLE
Document OIDC options for Spectro VM Dashboard: PEM-2283

### PR DESCRIPTION
**Pull request by ritawatson**
## Describe the Change

This PR adds a doc to Configure OIDC options ([Preview](https://deploy-preview-1526--docs-spectrocloud.netlify.app/vm-management/vm-packs-profiles/configure_OIDC)) that allow Palette to display the VM Dashboard. 

The PR also cleans up formatting after the Docusaurus migration.

Recently, descriptions were added to address how dashboard access is based on the OIDC option the user selects:
    
CleanShot 2023-09-26 at 17 09 30

## Review Changes

💻 [Preview](https://deploy-preview-1526--docs-spectrocloud.netlify.app/vm-management/vm-packs-profiles/configure_OIDC)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/PEM-2283)

### CODE REVIEW

**Comment by ritawatson** [comment]
This is work in progress - do not merge.

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+sidebar_label: "Configure OIDC"
+title: "Configure OIDC"
+description: "Learn how to configure OIDC so Palette displays the Spectro VM Dashboard."
```
Comment (cloudmaniac) [inline docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md:4]:
```suggestion
description: "Learn how to configure OIDC so Palette displays the Virtual Machine Dashboard."
```

**Review by cloudmaniac — APPROVED** [review]
LGTM, I only provided a single suggestion. I'll let @tiwarisumit3 answer your comment in PEM-2283.

File: docs/docs-content/vm-management/vm-packs-profiles/add-roles-and-role-bindings.md [hunk 2]
```diff
@@ -48,6 +43,8 @@ You must configure permissions for actions that users can perform on Virtual Mac
 
 3. Click on **Settings** and choose **RBAC** to add role bindings. Refer to [Create a Role Binding](/clusters/cluster-management/cluster-rbac#createrolebindings) for guidance. Refer to [VM User Roles and Permissions](/vm-management/vm-roles-permissions) for a list of Cluster Roles and equivalent Palette Roles.
 
+    <!-- If you have OpenID Connect (OIDC) configured at the Kubernetes layer of your cluster profile, you can create a role binding that maps individual users or groups assigned within the OIDC provider's configuration to a role. To learn more, review [Use RBAC with OIDC](https://docs.spectrocloud.com/integrations/kubernetes/#use-rbac-with-oidc). -->
```
Comment (karl-cardenas-coding) [inline docs/docs-content/vm-management/vm-packs-profiles/add-roles-and-role-bindings.md]: Let's add this back in.

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+icon: " "
+hide_table_of_contents: false
+sidebar_position: 0
+tags: ["vmo", "oidc"]
+---
+
+
+Palette displays the VM dashboard based on OpenID Connect (OIDC) Identity Provider options you enable in the Kubernetes layer of the cluster profile you will use for your Virtual Machine Orchestrator (VMO) cluster.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md:12]: This feels like a run-on sentence.

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+## Configure OIDC Options
+
+1. Log in to [Palette](https://console.spectrocloud.com/).
+
+2. From the left **Main Menu** click **Profiles**.
+
+3. Select the cluster profile to update. Palette displays profile details and the profile stack.
+
+4. Select the Kubernetes layer in the profile stack, and choose **None** or **Palette** as the OIDC Identity Provider.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md:28]: Is `Custom`  or `Inherit from Tenant` not a value?

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+6. Save your changes.
+
+
+## Validate
+
+1. From the left **Main Menu** click **Profiles**.
+
+2. Select the cluster profile you updated. Palette displays profile details and the profile stack.
+
+3. Select the Kubernetes layer. Palette displays the OIDC Identity Provider you selected - either **None** or **Palette**.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md:41]: Same question as above.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md (hunk 4 above)
Comment (ritawatson) [reply docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md:28]: @karl-cardenas-coding They are, but from what I understand you must choose either None or Palette in order to display the dashboard. That's why I asked about the one paragraph in the Add Roles and Role Bindings doc for VMO.

File: docs/docs-content/vm-management/vm-packs-profiles/add-roles-and-role-bindings.md (hunk 2 above)
Comment (ritawatson) [reply docs/docs-content/vm-management/vm-packs-profiles/add-roles-and-role-bindings.md]: Added back.

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md (hunk 5 above)
Comment (ritawatson) [reply docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md:41]: @karl-cardenas-coding In order to display the VM dashboard, the user must select either one of these options. It's why I asked if the other OIDC options still applied.

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md (hunk 3 above)
Comment (ritawatson) [reply docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md]: Revised.

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md (hunk 4 above)
Comment (ritawatson) [reply docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md]: Added all `Custom` and `Inherit from Tenant`.

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+1. From the left **Main Menu** click **Profiles**.
+
+2. Select the cluster profile you updated. Palette displays profile details and the profile stack.
+
+3. Select the Kubernetes layer. Palette displays the OIDC Identity Provider you selected - either **None**, **Palette**, **Inherit from Tenant**, or **Custom**.
+
+
+## Next Steps
+
+Now you are ready to create the VMO profile. Refer to [Create the VMO Profile](/vm-management/vm-packs-profiles/create-vmo-profile) for guidance.
```
Comment (karl-cardenas-coding) [inline docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md:58]:
```suggestion
You are now ready to create the VMO profile. Refer to [Create the VMO Profile](/vm-management/vm-packs-profiles/create-vmo-profile) for guidance.
```
I think this sounds a bit more simpler.

@ritawatson this URL also needs to be fixed.

File: docs/docs-content/vm-management/vm-packs-profiles/add-roles-and-role-bindings.md [hunk 7]
```diff
@@ -48,6 +38,8 @@ You must configure permissions for actions that users can perform on Virtual Mac
 
 3. Click on **Settings** and choose **RBAC** to add role bindings. Refer to [Create a Role Binding](/clusters/cluster-management/cluster-rbac#createrolebindings) for guidance. Refer to [VM User Roles and Permissions](/vm-management/vm-roles-permissions) for a list of Cluster Roles and equivalent Palette Roles.
 
+    If you have OpenID Connect (OIDC) configured at the Kubernetes layer of your cluster profile, you can create a role binding that maps individual users or groups assigned within the OIDC provider's configuration to a role. To learn more, review [Use RBAC with OIDC](https://docs.spectrocloud.com/integrations/kubernetes/#use-rbac-with-oidc).
```
Comment (karl-cardenas-coding) [inline docs/docs-content/vm-management/vm-packs-profiles/add-roles-and-role-bindings.md:41]:
```suggestion
    If you have OpenID Connect (OIDC) configured at the Kubernetes layer of your cluster profile, you can create a role binding that maps individual users or groups assigned within the OIDC provider's configuration to a role. To learn more, review [Use RBAC with OIDC](https://docs.spectrocloud.com/integrations/kubernetes/#use-rbac-with-oidc).
```
This URL is not correct. It needs to point to the file name.

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+
+
+
+
+Palette displays the Virtual Machine dashboard based on the OpenID Connect (OIDC) Identity Provider option that you select in the Kubernetes layer of the infrastructure profile.
+
+
+## Prerequisites
+
+- A configured infrastructure profile. For more information, review [Create a Cluster Profile](/cluster-profiles/task-define-profile).
```
Comment (karl-cardenas-coding) [inline docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md:19]: This URL needs to point to the file name.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
URLs need to be corrected, along with merge conflicts.

File: docs/docs-content/vm-management/vm-packs-profiles/add-roles-and-role-bindings.md (hunk 7 above)
Comment (ritawatson) [reply docs/docs-content/vm-management/vm-packs-profiles/add-roles-and-role-bindings.md:39]: Fixed

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md (hunk 6 above)
Comment (ritawatson) [reply docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md:58]: Fixed URL. Agree about wording and fixed it.

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md (hunk 8 above)
Comment (ritawatson) [reply docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md:19]: Fixed.

Comment (karl-cardenas-coding) [inline docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md:19]: This URL needs to be updated so it points to the desired file.

**Review by karl-cardenas-coding — APPROVED** [review]

File: docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md (hunk 8 above)
Comment (ritawatson) [reply docs/docs-content/vm-management/vm-packs-profiles/configure_OIDC.md]: Fixed

**Pull request merged**